app/Http/Controllers/CustomFieldsController.php

```diff
@@ -90,6 +90,10 @@ public function store(CustomFieldRequest $request)
             $field->format = e($request->get("format"));
         }
 
+        if($request->get("element") == "checkbox"){
+            $field->format = 'boolean';
```

Comment:
It's not really boolean though, is it? It's potentially an array, since multiple checkboxes can be checked for one custom field

Reply:
It's not. In the database it never was stored as a boolean, the format only function to validate the restrictions. This PR makes checkboxes only work with boolean values, so it doesn't need to be an array.

Reply:
But there isn't a new database field created for each of the options in the checkboxes, so storing a checkbox field like `_snipeit_is_snipe_rad_12` as a boolean for a checkbox doesn't make sense (unless I am missing something). The options for that checkbox could be:

[x]  Yes
[ ] No
[x] Sometimes

How would you store that as boolean data? 

We'd more likely need to store it as JSON and encode/decode it to display the boxes as checked or not and to display it in the list results.